cogs/birthday/birthday.py

```diff
@@ -309,7 +259,7 @@ async def list(self, ctx: Context):
         page.embed.colour = discord.Colour.red()
         await page.paginate()
 
-    @_birthday.command(name="unassign", aliases=["remove", "rm"])
+    @_birthday.command(name="unassign")
```

Comment:
I left the `unassign` command in case we wanted to remove the birthday role from a user, but not delete them from the list. Hope that's okay.

Reply:
Yeah that should be fine.